terraform/dev/main.tf

```diff
@@ -130,3 +130,8 @@ module "consumer" {
   lev_api_url                 = "https://${module.lev_api.service_url}"
   schedule                    = "cron(0,30 9-18 ? * MON-FRI *)"
 }
+
+module "securityhub" {
+  source = "../modules/security_hub"
+  region = data.aws_region.current.name
+}
```

Comment:
Would this make more sense in shared?

Reply:
Possibly, it depends on how we're setting up different accounts though. We'd need to enable it in prod too, as it's an account level thing.